examples/deferred_processing/automated_deferred_processing/README.md

```diff
@@ -0,0 +1,348 @@
+Deferred processing uses branches to prevent pipelines from triggering on every input commit.
+This example shows how to automate the movement of those branches.
+
+The example here triggers a job periodically, using a cron pipeline.
+The Makefile you'll find in this example,
+along with the explanations provided in this document,
+should give you a good start on implementing this in your Pachyderm cluster
+with or without authentication enabled.
+
+## Prerequisites
+
+Before you begin, you should understand deferred processing by reading the documentation
+and trying the [deferred processing example](../deferred_processing_plus_transactions).
+That example is used extensively here.
+
+If you're using a Pachyderm cluster with authentication activated,
+this example will show you how to create a Pachyderm authentication token,
+load the token into a Kubernetes secret provisioned through `pachctl`,
+and use `pod_patch` to mount the secret as a Kubernetes volume
+and create environment variables for use by the pipeline.
+If you are unfamiliar with those things,
+it may be helpful to refer to the following documentation as you work through the example.
+* [Pachyderm authentication documentation](https://docs.pachyderm.com/latest/enterprise/auth/)
+* [Kubernetes documentation on
+Secrets](https://kubernetes.io/docs/concepts/configuration/secret/),
+* [pachctl create secret](https://docs.pachyderm.com/latest/reference/pachctl/pachctl_create_secret/) command
+* [Pachyderm documentation on adding a volume to a pipeline](https://docs.pachyderm.com/latest/how-tos/mount-volume/) using [pod_patch](https://docs.pachyderm.com/latest/reference/pipeline_spec/#pod-patch-optional)
+* the [jq utility](https://stedolan.github.io/jq/manual/) for transforming json files in shell scripts
+
+You need to have Pachyderm v1.9.8 or later installed on your computer or cloud platform. 
+See [Deploy Pachyderm](https://docs.pachyderm.com/latest/deploy-manage/deploy/).
+
+It helps to have some understanding of Makefiles, 
+if you plan on using those in your infrastructure.
+Basic Unix shell scripting skills are assumed.
+
+## Pipelines
+
+This example uses the same DAG as in the deferred processing example,
+with the addition of a cron pipeline 
+for periodically moving the `dev` branch to `master`.
+
+For details on the deferred processing example DAG,
+please refer to [that example](../deferred_processing_plus_transactions).
+
+### Branch mover, without authentication
+
+The cron pipeline is called `branch-mover`. 
+It is configured to run, by default, every 1 minute, 
+per its tick input:
+
+```
+  "input": {
+    "cron": {
+        "name": "tick",
+        "spec": "@every 1m",
+        "overwrite": true
+    }
+  },
+```
+
+The code for moving the branch is very simple, 
+if the Pachyderm cluster does not have authentication activated.
+
+Using the official Pachyderm pachctl image, 
+the transform first updates the default `pachctl` config
```

Comment:
Not sure what `transform` exactly is. Maybe the `transformation code` is a better way of saying it?

Reply:
clarified it:  "the transform section of the pipeline spec"